test/sna/auth/mail.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Sna.Auth.MailTest do
```

Comment:
file name `test/sna/auth/mail_test.ex` ?`

Reply:
I think we are in a directy named test, and we are using the namespace only to define that this is the test for Sna.Auth.Mail. I don't think it is particularly useful to rename the file like that.